.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -50,3 +68,8 @@ You may additionally include an explanation.
 #### If NVDA add-ons are disabled, is your problem still occurring?
 
 #### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
+
+#### in case your issue is related to a browser (i.e. Chrome), did you test the behavior of NVDA also in other browsers (i.e. Firefox or Microsoft Edge)? What are your findings in other browsers?
```

Comment:
Use "e.g." instead "i.e."

```suggestion
#### in case your issue is related to a browser (e.g. Chrome), did you test the behavior of NVDA also in other browsers (e.g. Firefox or Microsoft Edge)? What are your findings in other browsers?
```

Reply:
I actually really meant "for example" using i.e. would gramatically limit the sentence to Chrome or Firefox or Edge. Here is an article on the correct use of i.e. and e.g.
https://www.dictionary.com/e/ie-vs-eg/

I.e. is more like "that means" and e.g. is "for example".